src/main/nav_create_simulated_image.py

```
@@ -904,9 +999,14 @@ def _delete_tab_by_index(self, kind: str, data_index: int) -> None:
         if kind == 'body':
             if 0 <= data_index < len(self.sim_params['bodies']):
                 del self.sim_params['bodies'][data_index]
+        elif kind == 'ring':
+            if 0 <= data_index < len(self.sim_params['rings']):
+                del self.sim_params['rings'][data_index]
         elif kind == 'star':
             if 0 <= data_index < len(self.sim_params['stars']):
                 del self.sim_params['stars'][data_index]
+        else:
+            assert False, f'Unknown kind: {kind}'
```

Ruff (0.14.11)

Assertion always fails, replace with `pytest.fail()`

(PT015)